src/packages/v4/hooks/useNativeTokenSurplus.ts

```diff
@@ -0,0 +1,20 @@
+import { NATIVE_TOKEN } from 'juice-sdk-core'
+import {
+  useJBContractContext,
+  useReadJbMultiTerminalCurrentSurplusOf,
+} from 'juice-sdk-react'
+
+/**
+ * @todo add to sdk
+ */
+export function useNativeTokenSurplus() {
+  const {
+    projectId,
+    contracts: { primaryNativeTerminal },
+  } = useJBContractContext()
+
+  return useReadJbMultiTerminalCurrentSurplusOf({
```

Comment:
All the read hooks come from the SDK. This hook (`useNativeTokenSurplus`) is an just an example of where we're wrapping one of the sdk hooks. But per the TODO, this hook can probably live in the SDK (it's a super common hook that many people could use).

Reply:
Copy